Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```
@@ -765,6 +752,250 @@
             - ReagentId: AmbuzolPlus
               Quantity: 5
 
+- type: entity
+  name: pill
+  suffix: haloperidol 10u
+  parent: Pill
+  id: PillHaloperidol
+  components:
+    - type: Pill
+      pillType: 10
+    - type: Sprite
+      state: pill11
+    - type: SolutionContainerManager
+      solutions:
+        food:
+          maxVol: 20
+          reagents:
+            - ReagentId: Haloperidol
+              Quantity: 10
```

```suggestion
          reagents:
          - ReagentId: Haloperidol
            Quantity: 10
```
yaml convention
same for the others below